samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/KMLLayers.razor

```diff
@@ -0,0 +1,53 @@
+﻿@page "/kmllayers"
```

Comment:
Is this page added to `navMenu.razor`? I don't see it.

Reply:
corrected that oversight